### PR TITLE
the product-white-list configuration files don't be loaded from S3 storage

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Comment by huihuiw01** [comment]
Could you  add the issue id on the top comment of this PR?

File: g11n-ws/modules/md-data-api-pgimpl/src/main/java/com/vmware/vip/messages/data/dao/pgimpl/PgProductApiImpl.java [hunk 1]
```diff
@@ -97,5 +97,13 @@ public String getVersionInfo(String productName, String version) throws DataExce
              }
         }
 
+   /**
+    * get the white list content from pg db
+    */
+    @Override
+    public String getWhiteListContent() throws DataException {
+        return null;
```
Comment (huihuiw01) [inline g11n-ws/modules/md-data-api-pgimpl/src/main/java/com/vmware/vip/messages/data/dao/pgimpl/PgProductApiImpl.java]: Why not implement it?

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java [hunk 2]
```diff
@@ -117,7 +118,7 @@ public void addInterceptors(InterceptorRegistry registry) {
 		 */
 
 		// Request Validation
-		InterceptorRegistration apival = registry.addInterceptor(new APIValidationInterceptor(bundleFileBasePath)).addPathPatterns("/**").excludePathPatterns(API.I18N_API_ROOT+"doc/**");
+		InterceptorRegistration apival = registry.addInterceptor(new APIValidationInterceptor(productService.getWhiteList())).addPathPatterns("/**").excludePathPatterns(API.I18N_API_ROOT+"doc/**");
```
Comment (huihuiw01) [inline g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java]: Can this implementation apply to both saas and on-prem scenario?

Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java]: yes

**Pull request merged**